CONVENTIONS.md

```diff
@@ -113,6 +114,14 @@ front of the coordinate (`velocityX`, not `xVelocity`).
 
 Name screen coordinates `x` and `y`, rather than `left` and `top`.
 
+## Javadocs
+
+Write full sentences for class, method and fields javadocs, starting with an uppercase and ending with a period. Use HTML tags such as `<p>` if the docs have several paragraphs.
```

Comment:
`full sentences` may be inaccurate because we often omit subjects from method javadocs.

From https://www.dictionary.com/browse/full-sentence :
> ..., as, in English, any sentence that contains **a subject** and a predicate; ...

Reply:
You're right, changed wording